src/signed.rs

```diff
@@ -55,7 +55,7 @@ pub trait Signable {
 /// `[PartiallyMultisigned<'a, T, MK>]`.
 #[derive(Clone, Debug, Decode, Encode)]
 pub struct UncheckedSigned<T: Signable, S> {
-    signable: T,
+    pub(crate) signable: T,
```

Comment:
I think Misza's changes add a getter for this, which is nicer than making the field (crate)public.